javascripts/discourse/templates/components/dc-menu.hbs

```diff
@@ -25,51 +13,35 @@
   <div class="dc-container-fluid">
     <div class="dc-row">
       <div class="dc-col">
-        <div class="dc-menu-card dc-menu-card--membership">
-          <a
-            class="dc-text-monospace dc-text-lg"
-            href="https://membership.debtcollective.org/"
-            target="_blank"
-          >
+        <a class="dc-menu-card dc-menu-card--membership" href="https://membership.debtcollective.org/" target="_blank">
+          <span class="dc-text-monospace dc-text-lg">
```

Comment:
Kinda the same principle than above. It would be better to avoid the nested node just for text proposes as we only have a single text. An anchor tag that has only a text within will be 'better' in terms of a11y than nesting things so I will prefer instead like;

```
<a class="dc-menu-card dc-menu-card--membership dc-text-monospace dc-text-lg" href="https://membership.debtcollective.org/" target="_blank">
    Donate
</a>
```